src/NUnitFramework/framework/nunit.framework.csproj

```diff
@@ -8,9 +8,7 @@
 
   <ItemGroup>
     <PackageReference Include="Microsoft.NETFramework.ReferenceAssemblies" Version="1.0.0" />
-  </ItemGroup>
-
-  <ItemGroup>
+    <PackageReference Include="Nullable" Version="1.3.1" PrivateAssets="all" />
```

Comment:
Oh, now I understand. We've swapped out our own "shape" and copy for this. I'm unsure what the decision process is for taking on new dependencies, but I do like your inclusion of `PrivateAssets` here.
@jnm2 I think you did the initial pass in making some things nullable. Can I ask you your thoughts on this?

Reply:
Ah, this one confused me. I had to go out to the NuGet package and read the description to understand, "This package does not contain any compiled binaries, but instead adds the attribute classes as C# source code to your project." A note as such in the PR would have been helpful.